controller/schema/schema.go

```diff
@@ -39,6 +39,7 @@ func Load(schemaRoot string) error {
 		schema := &jsonschema.Schema{Cache: schemaCache}
 		err = schema.ParseWithoutRefs(file)
 		if err != nil {
+			file.Close()
```

Comment:
@titanous true, just changed. Edited through Github Web UI, can this be squashed on merge?